spring-data-cassandra/src/main/java/org/springframework/data/cassandra/core/mapping/CassandraPersistentEntity.java

```diff
@@ -50,6 +52,12 @@
 	 */
 	CqlIdentifier getTableName();
 
+
+	/**
+	 * Returns the keyspace to which the entity shall be persisted.
+	 */
+	@Nullable CqlIdentifier getKeyspace();
```

Comment:
Let's turn this method's return type into `Optional` and rename it to `getKeyspaceName`